packages/@tinacms/schema-tools/src/util/hasDuplicates.ts

```diff
@@ -11,10 +11,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-export function hasDuplicates<T = any>(array: T[]) {
+export function hasDuplicates<T = any>(array: T[] | undefined) {
   if (!array) {
     return false
   } else {
     return new Set(array).size !== array.length
   }
 }
+
+/**
+ *
+ * @param array
+ * @returns False if the array is undefined or has no duplicates.
+ */
+export function findDuplicates<T = any>(
+  array: T[] | undefined
+): false | string {
```

Comment:
Should this return undefined instead of false if no duplicated are found?

Reply:
It can. I am just doing a IF check on them so it would not matter.

Reply:
Updated